manageiq-gems-pending.gemspec

```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "more_core_extensions",    "~>3.2"
   s.add_runtime_dependency "net-scp",                 "~>1.2.1"
   s.add_runtime_dependency "net-sftp",                "~>2.1.2"
-  s.add_runtime_dependency "nokogiri",                "~>1.6.8"
+  s.add_runtime_dependency "nokogiri",                "~>1.7.1"
```

Comment:
@carbonin I'm just following the existing pattern. If @jrafanie is ok with changing it, I'm ok with it.